admin/user.go

```diff
@@ -149,15 +143,17 @@ func (s *Service) InviteUserToOrganization(ctx context.Context, email, inviterID
 	return nil
 }
 
-func (s *Service) InviteUserToProject(ctx context.Context, email, inviterID, projectID, roleID, projectName, roleName string) error {
-	// Validate email address
-	_, err := mail.ParseAddress(email)
+func (s *Service) NotifyUserAdditionToOrganization(ctx context.Context, email, orgName, roleName string) error {
```

Comment:
This can be inlined but we will need to export `email` from `Service` in order to use it in `server` package